.github/workflows/update_rol_parameters.yml

```diff
@@ -0,0 +1,39 @@
+name: Update ROL Parameters
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  update-parameters:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.x'
+
+    - name: Prepare environment
+      run: |
+        python -m pip install --upgrade pip
+        chmod +x find_parameters.sh
```

Comment:
@gregvw I think this step will run the root of the Trilinos checkout, not in `packages/rol/rol_parameters`.

Reply:
Ah yes. This file should be removed.

Reply:
Removed file.